### PR TITLE
add(config): add chain override to cli flags

### PR DESCRIPTION
**Pull request by Zygimantass**
Simple change, allows to override chain name for use in subcommands `tx`, `tendermint`, etc.

Usage:
`--chain <chain_name_in_config>`

If `--chain` is not defined, it'll default to value in config

### CODE REVIEW

**Comment by jackzampolin** [comment]
Thank you for the PR @Zygimantass! Will check this out soon

**Comment by jackzampolin** [comment]
Add this to the current working branch

**Comment by jackzampolin** [comment]
@Zygimantass  you will also need to wire this up in the https://github.com/strangelove-ventures/lens/blob/main/cmd/config.go#L70 `GetDefaultClient` command. That function should now take a `cmd *cobra.Command` and you should pull the value of the flag to have it override the default if the flag has a value.

File: cmd/config.go [hunk 1]
```diff
@@ -206,6 +206,17 @@ func initConfig(cmd *cobra.Command) error {
 		config.cl[name] = cl
 	}
 
+	// override chain if needed
```
Comment (Zygimantass) [inline cmd/config.go:209]: This would override the chain in GetDefaultClient. Is there another reason to get the value directly from cmd there?

Comment (jackzampolin) [reply cmd/config.go:209]: Oh I didn't see this. Thats a cool way of doing it let me test it.

**Review by jackzampolin — APPROVED** [review]

**Pull request merged**